internal/pkg/term/selector/selector.go

```diff
@@ -265,21 +265,18 @@ func (s *Select) Environment(prompt, help, app string) (string, error) {
 	return selectedEnvName, nil
 }
 
-// EnvironmentWithNone fetches all the environments in an app and prompts the user to select one of the environments or None.
-func (s *Select) EnvironmentWithNone(prompt, help, app string) (string, error) {
+// EnvironmentWithAdditionalOptions fetches all the environments in an app and prompts the user to select one of the environments or additional options.
+// NOTE: `additionalOpt` shouldn't be an empty string.
+func (s *Select) EnvironmentWithAdditionalOptions(prompt, help, app string, additionalOpt string, moreAdditionalOpts ...string) (string, error) {
```

Comment:
Got a bit confused at this one. Why not just
```suggestion
func (s *Select) EnvironmentWithAdditionalOptions(prompt, help, app string, additionalOpt ...string) (string, error) {
```

Reply:
Indeed they can be combined. I wanted to have custom log information for "environmentwithadditionaloptions" but that no longer seems necessary now.